traffic_ops/traffic_ops_golang/login/logout_test.go

```diff
@@ -0,0 +1,159 @@
+package login
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import "context"
+import "net/http"
+import "net/http/httptest"
+import "testing"
+import "time"
+
+// import "github.com/apache/trafficcontrol/lib/go-log"
```

Comment:
nit remove ?

Also another nit should imports be in import block here and above?

Reply:
I don't like setting precedents for arbitrary stylistic changes without a style guide. As a Python programmer this style is most pleasing to me. I'm not saying I'd refuse to put them in blocks, just explaining my choice. If and when (hopefully when) there _is_ a style guide, I'll be more than happy to conform to it.
That comment should absolutely be removed, though.

Reply:
👍 completely agree on the style guide hence the comment being a nit/question and still approval of the PR.

FWIW per golangs _style guide_ they have the imports as a block https://github.com/golang/go/wiki/CodeReviewComments#imports